mmdet3d/core/bbox/coders/km3d_decoder.py

```diff
@@ -0,0 +1,498 @@
+import numpy as np
+import torch
+from torch.nn import functional as F
+
+
+def _nms(heat, kernel=3):
+    """Process heat with Non Maximum Suppression.
+git
+    Args:
+        heat (torch.Tensor): Predictions from model in shape (batch_size *
+            channels * height * width)
+        kernel (int): kernel size for max pooling
+
+    Returns:
+        torch.Tensor: batch_size * channels * height * width
```

Comment:
Describe the specific content of the returned variable like 'Heatmap after NMS' instead of just the shape of the tensor